### PR TITLE
Docs: Canvas visualization refactor

### LINKED ISSUE #95303: Docs: Canvas restructure

### PR DESCRIPTION
**Pull request by imatwawana**
This PR:

- Refactors the page adding tables to cover short content and link to longer content.
- Adds missing options.
- Orders options to match the UI.
- Removes and replaces images.
- Fixes links.
- Fixes linting errors and warnings.
- Makes necessary wording edits/fixes typos.

**OUT OF SCOPE**: Style and copy edits

### CODE REVIEW

**Review by irenerl24 — APPROVED** [review]
Excellent work! Thank you for this!

File: docs/sources/panels-visualizations/visualizations/canvas/index.md [hunk 1]
```diff
@@ -253,22 +433,34 @@ The first data link in the list will be configured as your one-click data link.
...
 
-{{< docs/shared lookup="visualizations/thresholds-options-2.md" source="grafana" version="<GRAFANA_VERSION>" >}}
+### Standard options
 
-## Value mappings
+{{< docs/shared lookup="visualizations/standard-options.md" source="grafana" version="<GRAFANA_VERSION>" >}}
+
+### Value mappings
 
 {{< docs/shared lookup="visualizations/value-mappings-options.md" source="grafana" version="<GRAFANA_VERSION>" >}}
```
Comment (irenerl24) [inline docs/sources/panels-visualizations/visualizations/canvas/index.md:458]: worth noting that in the preview nothing appears here

File: docs/sources/panels-visualizations/visualizations/canvas/index.md [hunk 2]
```diff
@@ -253,22 +433,34 @@ The first data link in the list will be configured as your one-click data link.
...
 
-## Field overrides
+### Field overrides
 
 {{< docs/shared lookup="visualizations/overrides-options.md" source="grafana" version="<GRAFANA_VERSION>" >}}
```
Comment (irenerl24) [inline docs/sources/panels-visualizations/visualizations/canvas/index.md:462]: worth noting that in the preview nothing appears here

File: docs/sources/panels-visualizations/visualizations/canvas/index.md [hunk 3]
```diff
@@ -253,22 +433,34 @@ The first data link in the list will be configured as your one-click data link.
...
+
+### Thresholds
+
```
Comment (irenerl24) [inline docs/sources/panels-visualizations/visualizations/canvas/index.md:465]: worth noting that in the preview nothing appears here

**Review by irenerl24 — APPROVED** [review]

File: docs/sources/panels-visualizations/visualizations/canvas/index.md (hunk 1 above)
Comment (imatwawana) [reply docs/sources/panels-visualizations/visualizations/canvas/index.md:458]: Thanks for the heads up. It works in the local preview, but I've raised this with the docs platform team.

**Pull request merged**